bigbluebutton-html5/imports/ui/components/presentation/presentation-toolbar/component.jsx

```diff
@@ -118,14 +118,17 @@ class PresentationToolbar extends Component {
     // Fill drop down menu with all the slides in presentation
     const optionList = [];
     for (let i = 1; i <= numberOfSlides; i += 1) {
-      optionList.push(
+      optionList.push((
         <option
           value={i}
           key={i}
         >
-        Slide {i}
-        </option>,
-      );
+          <FormattedMessage
```

Comment:
We don't use this method for localization. 
/imports/ui/components/nav-bar/component.jsx is a good example of how it is supposed to be done.
[define](https://github.com/bigbluebutton/bigbluebutton/blob/v2.0.x-release/bigbluebutton-html5/imports/ui/components/nav-bar/component.jsx#L19) messages -> use [intl.formatMessage](https://github.com/bigbluebutton/bigbluebutton/blob/v2.0.x-release/bigbluebutton-html5/imports/ui/components/nav-bar/component.jsx#L176) to inject your strings -> use [injectIntl](https://github.com/bigbluebutton/bigbluebutton/blob/v2.0.x-release/bigbluebutton-html5/imports/ui/components/nav-bar/component.jsx#L201) at the end.

`<FormattedMessage>` was added by mistake a long time ago, and nobody came across this file to change it. I'm not sure if it even works 😄

Reply:
@OZhurbenko  They are the same, aren't they?

> This component uses the formatMessage API and has props that correspond to a Message Descriptor.

https://github.com/yahoo/react-intl/wiki/Components#formattedmessage

Reply:
> By default <FormattedMessage> will render the formatted string into a <span>.

Well, sometimes we need to inject an aria-label / aria-labelledby / aria-describedby attributes right into the html element. Sometimes I can see we use `<label htmlFor={"...">` instead of `span`

I guess it may work in this particular case, but we'll need to change it to our common way either now or later anyway.

Reply:
Even if they were identical, we still need to choose one version and stick with it everywhere.